synapse/storage/data_stores/main/events.py

```diff
@@ -61,6 +62,13 @@
     ["type", "origin_type", "origin_entity"],
 )
 
+STATE_EVENT_TYPES_TO_MARK_UNREAD = [
+    EventTypes.PowerLevels,
+    EventTypes.Topic,
+    EventTypes.Name,
+    EventTypes.RoomAvatar,
+]
```

Comment:
let's make this a set for efficient `in` performance:

```suggestion
STATE_EVENT_TYPES_TO_MARK_UNREAD = {
    EventTypes.PowerLevels,
    EventTypes.Topic,
    EventTypes.Name,
    EventTypes.RoomAvatar,
}
```

see: https://github.com/matrix-org/synapse/issues/7737